templates/terraform/.github/workflows/auto-format.yml

```diff
@@ -6,7 +6,7 @@ on:
 jobs:
   auto-format:
     runs-on: ubuntu-latest
-    container: cloudposse/build-harness:slim-latest
+    container: cloudposse/build-harness:latest
```

Comment:
@maximmi @goruha Since the `auto-format.yml` is installed in every Terraform repo, and we do not currently have an automated way of updating it, this change (or the need for this change) should have alerted you to the fact that committing this PR would break auto-format in every repo.